src/contactTab.js

```diff
@@ -0,0 +1,43 @@
+import avatar from './assets/avatar.jpg'
+
+function contactTab() {
```

Comment:
- [ ] As this project introduces ES6, all functions should be [arrow functions](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/Arrow_functions), kindly modify them before merging.